CHANGELOG.md

```diff
@@ -18,6 +18,27 @@
 
 ---
 
+## v5.28.0
+
+**Public Changes:**
+
+- ✨replace react-router wildcard routes with their actual path name ([#3023](https://github.com/DataDog/browser-sdk/pull/3023)) [RUM-REACT]
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏Missing space before `replace`